tests/test_runlib.py

```diff
@@ -554,6 +585,44 @@ def test_in_toto_wrong_key(self):
       in_toto_run(self.step_name, None, None,
           ["python", "--version"], True, self.key_pub)
 
+  def test_nonexistent_directory(self):
+    """Fail run, passed metadata_directory not exist. """
+    expected_error = IOError if sys.version_info < (3, 0) \
+        else FileNotFoundError
+    with self.assertRaises(expected_error):
+      in_toto_run(self.step_name, None, None, ["python", "--version"],
+          True, self.key, metadata_directory='nonexistentDir')
+
+  @unittest.skipIf(sys.version_info < (3, 0),
```

Comment:
This makes some sense to me, but are we never checking for `IOError` on Python 2?